### PR TITLE
Notification recipients

### PR DESCRIPTION
**Pull request by adambraimbridge**
cc @andygout 

Re: https://github.com/Financial-Times/next-flag-expiry-notifier/issues/12

image

### CODE REVIEW

File: test/app.spec.js [hunk 1]
```diff
@@ -115,7 +137,8 @@ describe('App', function(){
 			.end(done);
 	});
 
-	it('Should be able to activate a flag', function(done){
+	// This test is really tricky when changing the flag schema. Chicken and egg situation.
+	it.skip('Should be able to activate a flag', function(done){
```
Comment (adambraimbridge) [inline test/app.spec.js]: I'll remove this `skip` once ... actually ... yeah I'll fix it now

**Comment by andygout** [comment]
👍  Nice.

**Comment by adambraimbridge** [comment]
- I changed my mind after merging into master.
- I cancelled the master build in CI  (so production is unaffected)
- I've reverted the merge to master.

Before merging, I will need to be ready to update all of the flags to use the new `notificationRecipients` field. Otherwise, if someone else tries to edit a flag, it'll fail, because they won't know about `notificationRecipients` yet.

**Comment by andygout** [comment]
Roger that and good thinking.

**Pull request merged**